package.json

```diff
@@ -13,7 +13,7 @@
     "url": "https://github.com/AnomalyInnovations/serverless-nodejs-starter.git"
   },
   "devDependencies": {
-    "aws-sdk": "^2.633.0",
+    "aws-sdk": "^2.636.0",
```

Comment:
Another tech task... We should setup [dependabot](https://dependabot.com/) to automatically update dependencies for us.